sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -314,21 +315,27 @@ case class DropNamespace(
  */
 case class DescribeNamespace(
     namespace: LogicalPlan,
-    extended: Boolean,
-    override val output: Seq[Attribute] = DescribeNamespace.getOutputAttrs) extends UnaryCommand {
+    extended: Boolean) extends UnaryCommand {
+  override val output: Seq[Attribute] = {
```

Comment:
We shouldn't do this, as the output expr ID becomes unstable and it changes after the plan is copied.

Reply:
OK.